spec/javascripts/improve-this-page-spec.js

```diff
@@ -0,0 +1,207 @@
+describe("Improve this page", function () {
+   var FIXTURE =
+    '<div class="improve-this-page">' +
+      '<div class="js-prompt">' +
+        '<span class="improve-this-page__is-useful-question">Is this page useful?</span>' +
+        '<a href="/contact/govuk" class="js-page-is-useful" data-track-category="improve-this-page" data-track-action="page-is-useful">Yes</a>' +
+        '<a href="/contact/govuk" class="js-offer-feedback">No</a>' +
+        '<div class="improve-this-page__anything-wrong">' +
+          '<a href="/contact/govuk" class="js-offer-feedback">Is there anything wrong with this page?</a>' +
+        '</div>' +
+      '</div>' +
+      '<div class="js-feedback-form js-hidden" data-track-category="improve-this-page" data-track-action="give-feedback">' +
+        '<div class="js-errors"></div>' +
+        '<form>' +
+          '<input type="hidden" name="url" value="http://example.com/path/to/page"></input>' +
+          '<input type="hidden" name="user_agent" value="Safari"></input>' +
+          '<div>' +
+            '<label>How should we improve this page?</label>' +
```

Comment:
The markup in the fixture doesn't match the markup you are using. 
You've got labels wrapping the label text, hint text and input field - but not here.